.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -386,6 +404,17 @@ iam_CreateAccessKey:
   synopsis: create an &IAM; access key.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateAccessKeyExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateAccessKey
```

Comment:
These two examples are almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
removed the first example